.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,29 @@
-**Pull request recommendations:**
-- [ ] Name your pull request _your-development-type/short-description_. Ex: _feature/read-tiff-files_
-- [ ] Link to any relevant issue in the PR description. Ex: _Resolves [gh-12], adds tiff file format support_
-- [ ] Provide context of changes.
-- [ ] Provide relevant tests for your feature or bug fix.
-- [ ] Provide or update documentation for any feature added by your pull request.
-
-Thanks for contributing!
\ No newline at end of file
+<!--
+  Thank you for submitting a pull request!
+
+  ⚠️⚠️ Please do the following before submitting: ⚠️⚠️
+
+  - Read the CONTRIBUTING.md guide and make sure you've followed all the steps given.
+  - Ensure that the code is up-to-date with the `main` branch.
+  - Provide or update documentation for any feature added by your pull request.
+  - Provide relevant tests for your feature or bug fix.
+
+  ❗️ Also: ❗️
+
+  Please name your pull request {development-type}/{short-description}.
+  For example: feature/read-tiff-files
+-->
+
+### Link to Relevant Issue
+
+This pull request resolves #
+
+### Description of Changes
+
+_Include a description of the proposed changes._
+
+### Link to Forked Storybook Site
+
+_If component changes (especially visual changes) are contained in this PR, we ask that you provide a link to a publicly viewable version of the Storybook docs site, so PR reviewers can see your changes without having to install and view your code locally._
```

Comment:
Screenshots are also OK here.

Reply:
I slightly disagree personally. I want to see the component work / load on _any_ browser rather than just a screenshot personally. :shrug: 

That said, it's minor and I wouldn't care too much if someone dropped screenshots.

Reply:
Yeah, screenshots for minor changes are acceptable. I like to preview the changes on any browser too.
But in the case where a contributor has multiple PRs open, they might not be able to have a gh-page deployment for each PR.

Reply:
Very fair point on that one.

Reply:
Overall, provide a link to gh-page if you can. If not, then provide screenshots.